implementation/src/test/java/io/smallrye/mutiny/operators/MultiOnEventTest.java

```diff
@@ -160,8 +199,8 @@ public void testCallbacksOnFailure() {
                 .onCompletion().invoke(() -> completion.set(true))
                 .onTermination().invoke((f, c) -> termination.set(f != null))
                 .onTermination().invoke(() -> termination2.set(true))
-                .on().request(requests::set)
```

Comment:
As this is testing deprecated APIs, shouldn't it be using `on().request()`?

Or did I misunderstand which API is being deprecated?

Reply:
`on().request(action)` is being deprecated, but not `on().request()` which opens to the group

Reply:
So you can do:

`.onRequest().invoke(action)` and `.on().request().invoke(action)`

Reply:
This brings everything in line with the recent reworks (see completion / termination / etc)

Reply:
Right, but the test is testing deprecated APIs, based on the test name, so shouldn't it continue using `on().request(action)` instead of using a non-deprecatedAPI?

Reply:
`testCallbacksOnFailure` is not testing deprecated APIs, or am I missing the code you are looking at?

Reply:
Sorry, I'm confused from the view as I see `testCallbacksWhenItemIsEmittedWithDeprecatedApis()` as code and kept missing `testCallbacksOnFailure` in the smaller print